ClothesApp/ClothesApp/Program.cs

```diff
@@ -1 +1,225 @@
-﻿Console.WriteLine("Hello");
\ No newline at end of file
+﻿using System.Collections;
+using ClothesApp;
+using ClothesApp.Entities;
+using Microsoft.EntityFrameworkCore;
+
+using var db = new ClothesAppContext();
+
+Console.WriteLine("-----------------LAZY-----------------------");
+Console.WriteLine("get_products_by_brand_id");
```

Comment:
Please move all these calls to a separate methods, at this moment program file is a mess